src/main/java/io/strimzi/kafka/quotas/VolumeUsage.java

```diff
@@ -31,46 +31,47 @@ public VolumeUsage(String brokerId, String logDir, long capacity, long available
     }
 
     /**
-     *
+     * The source BrokerId.
      * @return The brokerId for the broker holding the volume
      */
     public String getBrokerId() {
         return brokerId;
     }
 
     /**
-     *
+     * The log directory, logDir in Kafka parlance.
      * @return the path identifying the logDir on the broker its hosted by.
      */
     public String getLogDir() {
         return logDir;
     }
 
     /**
+     * The capacity of the underlying Volume.
      * @return The size of the volume in bytes.
      */
     public long getCapacity() {
         return capacity;
     }
 
     /**
-     *
+     * The available capacity of the underlying volume.
      * @return The number available (free) remaining on the volume.
      */
     public long getAvailableBytes() {
         return availableBytes;
     }
 
     /**
-     *
+     * The consumed capacity of the underlying volume.
```

Comment:
Is this called from anywhere? If not should we remove it?

Reply:
Same answer as above we will need it for the metrics at least and potentially for other future limit policies. 

I'm somewhat on the fence if including for future policies is a good choice or not...